seed/static/seed/partials/insights_property.html

```diff
@@ -0,0 +1,159 @@
+<div class="page_header_container" ng-cloak>
+    <div class="page_header">
+        <div class="left page_action_container"></div>
+        <div class="page_title"><h1 translate>Property Insights</h1></div>
+        <div class="right page_action_container"></div>
+    </div>
+</div>
+
+<div class="section_nav_container">
+    <div class="section_nav" ng-include="::urls.static_url + 'seed/partials/insights_nav.html'"></div>
+</div>
+
+
+<div class="section_content_container">
+  <div class="section_content container-fluid">
+    <!-- <div class="content_block row" style="padding:20px 20px 0px 20px">
+      <div class="col-md-12">
+        <h2 style="font-size:18px;"> {$:: 'Property Insights' | translate $}</h2>
+      </div>
+    </div> -->
+    <div class="content_block row less_pad">
+
+    	<div ng-show="!data">
+				<div class="col-md-10">
+          <div class="title-row">
+              <h2 style="font-size:18px;"> {$:: 'Property Insights' | translate $}</h2>
+          </div>
+          <div class="compliance-setup">
+            <p>If you are not seeing a chart on this page, visit the <a ui-sref="program_setup(::{organization_id: organization.id})">Program Metrics</a> page to configure your program's metrics.
+            </p>
+          </div>
+      	</div>
+    	</div>
+    	<div ng-show="data">
+
+	    	<div class="col-md-2">
+	    		<div class="title-row">
+	        	<h2 style="font-size:18px;"> {$:: 'Property Insights' | translate $}</h2>
+	      	</div>
+	    			<!-- chart setup form-->
+					  <div class="form-group">
+					    <label for="cycleSelect">Cycle</label>
+					    <select class="form-control" id="cycleSelect" ng-model="chart_cycle" ng-change="update()" ng-options="operator.id as operator.name for operator in cycles">
+					    </select>
+					  </div>
+
+					  <!-- metric and X axis -->
+					  <div class="form-group">
+					    <label for="metricSelect">Metric Type</label>
+					    <select class="form-control" id="metricSelect" ng-model="chart_metric" ng-change="update()" ng-options="operator.id as operator.name for operator in y_axis_options">
+	 				    </select>
+					  </div>
+					  <div class="form-group">
+					    <label for="xSelect">X Axis</label>
+					    <select class="form-control" id="xSelect" ng-model="chart_xaxis" ng-change="update()" ng-options="operator.id as operator.display_name for operator in x_axis_options">
+	 				    </select>
+					  </div>
+
+	    	<!--<div class="margin-top-20">
```

Comment:
Remove?

Reply:
I think this is a placeholder for the export functionality. No harm in keeping it.